src/um/wincon.rs

```diff
@@ -538,3 +538,4 @@ extern "system" {
         ExeNameBufferLength: DWORD,
     ) -> DWORD;
 }
+pub type HPCON = *mut c_void;
```

Comment:
This is actually defined in `wincontypes.h`

Reply:
I can move it to a new file "wincontypes.rs". Do I need to add an accompanying feature for this new file?

I'd originally not done so because it seemed a bit overkill to make a new file for just the one line! While I'm at it want me to attempt to wrap whatever else is defined in `wincontypes.h`?

Reply:
You would need to add a feature for the new file. It's up to you whether you want to bind whatever else is in there.

Reply:
Great - will see how much I can find time for this weekend.